New resource: tfe_admin_organization_settings (#709)
hashicorp/terraform-provider-tfe: CHANGELOG.md

## Unreleased

DEPRECATIONS:

> * r/tfe_sentinel_policy is deprecated in favor of a new resource tfe_policy, which supports both Seninel and OPA policies

Reviewer: ```suggestion
* r/tfe_sentinel_policy is deprecated in favor of the new resource `tfe_policy`, which supports both Sentinel and OPA policies
```

> * r/tfe_organization_module_sharing is deprecated in favor of a new resource tfe_admin_organization_settings, which supports the global module sharing option

Reviewer: 💅 
```suggestion
* r/tfe_organization_module_sharing is deprecated in favor of the new resource `tfe_admin_organization_settings`, which supports the global module sharing option
```
